libraries/client/client_webhook.py

```diff
@@ -388,6 +400,31 @@ def add_payload_to_tx_converter(self, callback_url, identifier, payload, rc, sou
                 job = json_data['job']
         return identifier, job
 
+    def send_lint_request_to_tx_manager(self, job, repo_url):
+        payload = {
+            'job_id': job['job_id'],
+            'resource_type': job['resource_type'],
+            'file_type': job['input_format'],
```

Comment:
Ah, good point. How about this, in addition to my new addition to send the commit_data (for reference to the DCS repo and commit) and have that available in self.commit_data, I will also send the RC data and have that available in self.rc